portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inconsolata&family=Roboto:wght@300;400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+        <div id = "intro">
+            <p>Hi I'm</p>
+            <div id="name">
```

Comment:
If the purpose of this div is simply for styling reasons, just set the ID on the h2 itself.